types/ari-client/paackage.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
typo in filename

Reply:
This typo `paackage` seems to be causing the build to fail

Reply:
oh, yes, thank you!